src/query/xasl.h

```diff
@@ -949,6 +952,23 @@ struct partition_spec_node
 };
 #endif /* defined (SERVER_MODE) || defined (SA_MODE) */
 
+struct sq_cache
```

Comment:
I think it is better to have it defined in the src/query/subquery_cache.h file.

Reply:
I've applied it.